packages/client/.env.example

```
@@ -1 +1,5 @@
-CLIENT_TEST_WALLET_PRIVATE_KEY=
\ No newline at end of file
+TESTING_ENV_URL=
+TESTING_HEADER_KEY=
+TESTING_HEADER_VALUE=
+
+
```

```suggestion
TESTING_ENV_URL=
TESTING_HEADER_KEY=
TESTING_HEADER_VALUE=
```

Removed two extra lines at last

---

done!